src/common/utils/getApproveErrorMessage.js

```diff
@@ -8,8 +8,8 @@ export const APPROVE_ERROR_CODES = {
   inactiveExpenseClass: 'inactiveExpenseClass',
 };
 
-export const getApproveErrorMessage = (errorCode, genericError = 'ui-invoice.invoice.actions.approve.error') => {
+export const getApproveErrorMessage = (errorCode, genericError = 'ui-invoice.invoice.actions.approve.error', action = 'approve') => {
```

Comment:
does it make sense to change function name?

Reply:
any ideas?

Reply:
mb `getInvoiceActionErrorMessage` or `getActionErrorMessage` as it's already not only for approve